client/view/workbook/info/info.html

```diff
@@ -0,0 +1,50 @@
+<div class="container">
+
+
+    <header>
+        <h2>{workbook.name}</h2>
+    </header>
+
+
+    <main>
+        {#if forgotten}
+            {workbook.name} has been deleted.<br>
+            <button style="background-color: var(--green);" on:click="restoreWorkbook({key}, {workbook})">Restore</button>
```

Comment:
Right now the default button styles (in `input.css`) are used on the home screen as the primary/action button, we should probably keep that behavior here and not add any extra colors.  Though at some point eventually we'll probably want to add explicit classes for primary/cancel/whatever button styles.

```suggestion
            <button on:click="restoreWorkbook({key}, {workbook})">Restore</button>
```